(DOCSP-26079): Update Dart Standalone/Flutter SDKs (#2412)
mongodb/docs-realm: source/sdk/flutter/install.txt

      .. code-block:: dart

        import 'package:realm_dart/realm.dart';

.. _flutter-update-package:

Update Package Version
----------------------

> If you would like to use a version of the Flutter SDK or Dart Standalone SDK

Reviewer: The wording on this just feels a little awkward to me. Could we say something simpler like:

"To change the version of the Flutter SDK or Dart Standalone SDK in your project..."

> If you would like to use a version of the Flutter SDK or Dart Standalone SDK
> different from the one currently in your project, follow these steps.

Author: ```suggestion
To change the version of the Flutter SDK or Dart Standalone SDK
in your project, follow these steps.
```

.. tabs::

   .. tab:: Flutter
      :tabid: flutter

      #. Add the new SDK version to your :file:`pubspec.yaml` file.

         .. code-block:: yaml
           :caption: pubspec.yaml

           dependencies:
             realm: <updated_version>

      #. Install the updated version.

         .. code-block::

            flutter pub upgrade realm

      #. Regenerate your Realm Object models.

         .. code-block::

>             dart run realm_dart generate

Reviewer 2: ```suggestion
            flutter pub run realm generate
```

Reviewer 3: sorry no.. you could run:
```sh
dart run realm generate
```
as well

Reviewer 3: .. anyway better go with @desistefanova suggestion..

Author: thanks! this was def some bad copy pasting on my part
